step.yml

```diff
@@ -0,0 +1,54 @@
+
+title: |
+  Create ZIP
+summary: |
+  Create ZIP
```

Comment:
Please include a very little more of a description (and the same in description field below), for example: Creates a zip file to the given path with the file/dir given. (or some more nice sentence)